env.default

```diff
@@ -0,0 +1,15 @@
+# Wagtail config
+
+ALLOWED_HOSTS=*
+ASSET_DOMAIN=donate.mofoprod.net
+CONTENT_TYPE_NO_SNIFF=True
+CORS_WHITELIST=*
+CORS_REGEX_WHITELIST=""
+DATABASE_URL=""
```

Comment:
Should we add a default here (e.g. `postgres://donate_wagtail:donate_wagtail@localhost:5432/donate_wagtail`)?

If we did, I suppose we'd also have to add `environment` vars for the `postgres` service in `docker-compose.yml`, e.g:

```
postgres:
    image: postgres:11
    ports:
      - "5432"
    environment:
      - POSTGRES_USER=donate_wagtail
      - POSTGRES_PASSWORD=donate_wagtail
      - POSTGRES_DB=donate_wagtail
    volumes:
      - postgres_data:/var/lib/postgresql/data/:delegated

Reply:
Or we could keep it simple and stick with the defaults? e.g.
`postgres://postgres:postgres@localhost:5432/postgres)`

Reply:
The default currently is to use sqlite in the absence of `DATABASE_URL`, which I think is the most cross-platform compatible option. Mozillians develop on various OS/platform combos and I assumed this is why the default environment defaults to sqlite.

Reply:
(note that docker is not the standard development environment, just one option that we support - see http://github.com/mozilla/foundation.mozilla.org/ for the options they use there).

Reply:
> Mozillians develop on various OS/platform combos and I assumed this is why the default environment defaults to sqlite.

Ah, Okay. sqllite seems like a sensible default in that case :)